pkg/protostore/protostore.go

```diff
@@ -413,12 +443,32 @@ func (p ProtobufDataStore) FindAll(ctx context.Context, msgs interface{}, pagina
 	return metadataMap, nil
 }
 
-func (p ProtobufDataStore) SoftDeleteById(ctx context.Context, id string, msg proto.Message) (int64, error) {
+// FindAll Finds all messages (of the same type as the element of msgs) in Protostore and stores the result in msgs.
+// msgs must be a pointer to a slice of Protobuf structs or a pointer to a slice of pointers to Protobuf structs.
+// It will be modified in-place.
+// Returns a map of Protobuf messages' IDs to their metadata (parent ID & revision).
+func (p ProtobufDataStore) FindAll(ctx context.Context, msgs interface{}, pagination *datastore.Pagination) (metadataMap map[string]Metadata, err error) {
+	return p.findAll(ctx, msgs, pagination, false)
```

Comment:
use findAll() helper function to reuse common code between FindAll() and FindAllIncludingSoftDeleted()